oarepo_ui/theme/webpack.py

```diff
@@ -58,6 +58,7 @@
     dependencies = {
         "react-datepicker": "^4.21.0",
         "edtf": "^4.0.0",
+        "clipboard": "^2.0.11",
```

Comment:
No need for external dependency, see here: https://github.com/inveniosoftware/invenio-app-rdm/blob/09c8d76741f48e2dfee7ea670e68972e12577688/invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/ShareOptions/AccessLinks/LinksSearchItem.js#L73 https://caniuse.com/clipboard